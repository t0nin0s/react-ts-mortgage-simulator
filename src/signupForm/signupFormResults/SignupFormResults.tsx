import * as React from 'react';
import Card from '../../components/Card/Card';
const styles = require('../signupForm.css');

interface SignupFormResults {
  title: string;
  name: string;
  surname: string;
  resident: string;
  gender: string;
}

const SignupFormResults: React.SFC<SignupFormResults> = (props) => {
  return (
    <Card
      formTitle="Sign up"
      icon="handshake-o"
      footer={true}
    >
      <div className={styles.form__question}>
        Congratulations! Below is your data
      </div>
      <div className={styles.form__body}>
        <pre>
          {JSON.stringify(props)}
        </pre>
      </div>
      <div className={styles.form__submit} />
    </Card>
  );
};

export default SignupFormResults;
