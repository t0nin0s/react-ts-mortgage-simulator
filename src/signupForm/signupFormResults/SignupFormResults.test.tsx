import * as React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import SignupFormResults from './SignupFormResults'

describe('SignupFormResident renders as expected', () => {
  it('should render as expected', () => {
    const component = shallow(
      <SignupFormResults
        title='Mr'
        name='David'
        surname='Perez'
        gender='Male'
        resident='Yes' />
    )
    const tree = toJson(component)
    expect(tree).toMatchSnapshot()
  })
})
