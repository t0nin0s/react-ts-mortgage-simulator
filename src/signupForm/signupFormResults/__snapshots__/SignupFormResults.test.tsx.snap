// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignupFormResident renders as expected should render as expected 1`] = `
<Card
  footer={true}
  formTitle="Sign up"
  icon="handshake-o"
>
  <div>
    Congratulations! Below is your data
  </div>
  <div>
    <pre>
      {"title":"Mr","name":"David","surname":"Perez","gender":"Male","resident":"Yes"}
    </pre>
  </div>
  <div />
</Card>
`;
